data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/remedial_action/range_action/ComplexRangeAction.java

```diff
@@ -31,43 +31,31 @@
     public static final double TEMP_MAX_VALUE = 0;
 
     @JsonProperty("ranges")
```

Comment:
we need to decide if we want to delete this class or to keep it and use it to represent a rangeaction with several network elements.

Reply:
if we want to keep this class we have to rename it in order to avoid confusion with complexnetworkaction

Reply:
I decided to keep this class, rename it to AlignedRangeAction. It now represents a rangeaction with several network elements. I adapted the java doc and test accordingly